src/stories/components/custom-table/list-core-unit/list-core-unit.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { columns, CoreUnit } from '@ses/core/utils/test.utils';
+
+import ListCoreUnit from './list-core-unit';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/CUTable/ListCoreUnit',
+  component: ListCoreUnit,
+  parameters: {
+    chromatic: {
+      viewports: [1194, 1280, 1440],
+      pauseAnimationAtEnd: false,
+    },
+  },
+} as ComponentMeta<typeof ListCoreUnit>;
+
+const variantsArgs = [
+  {
+    queryStrings: '',
+    isLoading: false,
+    columns,
+    rows: [
+      {
+        value: CoreUnit,
+        key: CoreUnit.code,
+      },
+      {
+        value: CoreUnit,
+        key: `${CoreUnit.code}+1`,
+      },
+      {
+        value: CoreUnit,
+        key: `${CoreUnit.code}+2`,
+      },
+      {
+        value: CoreUnit,
+        key: `${CoreUnit.code}+3`,
+      },
+    ],
+  },
+];
+export const [[Summary, SummaryDark]] = createThemeModeVariants(ListCoreUnit, variantsArgs);
```

Comment:
please, use other names here as this is not related with summary

Reply:
done